test-utils/generators/Generators.scala

```diff
@@ -80,6 +80,11 @@ trait Generators
       .suchThat(!_.isValidInt)
       .map(_.formatted("%f"))
 
+  def intsInRangeWithCommas(min: Long, max: Long): Gen[String] = {
```

Comment:
Delete this. Not needed and we should not upset the Int behaviour.

Reply:
done